card/VERSION

```diff
@@ -1 +1 @@
-1.14.8s
+1.15.pre
```

Comment:
The name of the repo base is still something we should be in sync on because it is needed in the tools. (at least for now, we can move to individual roots if we find a good way to do that at some point)